.github/ISSUE_TEMPLATE/app_rendering_bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Application Card Rendering Bug Report
+about: Create a bug report for problems rendering Adaptive Cards in an Application
+title: "[Rendering] Bug Title Here"
+labels: Bug Triage-Needed Rendering
+assignees: ''
+---
+
+# Target Platform
+
+Please list the platform(s) that your application is running on (ex: Web, iOS, Android, etc)
+
```

Comment:
The previous template also asked hosts to identify themselves. Helpful for identifying bugs from known partners (Teams, Cisco)

Reply:
Ah good idea, I'll add that

Reply:
I don't see target app in this template. It's relevant for renderer bugs only because we prioritize bugs from known hosts like Teams/Cisco.

Reply:
Ahhh gotcha, sorry I misunderstood. I'll add that in